setup-without-nix.sh

```diff
@@ -109,6 +111,14 @@ EOF
       ~ephemeraladmin/.ssh/id_ed25519 \
       ~ephemeraladmin/.ssh/id_ed25519.pub
 
+    mkdir -p /var/lib/buildkite-agent/hooks
+    cat <<EOF > /var/lib/buildkite-agent/hooks/agent-shutdown
+#!/bin/sh
+
+curl -v http://bonk
```

Comment:
Do we want to log this output anywhere, or will it show up in the buildkite-agent log?

Reply:
It'll show up in the buildkite-agent log.

I think we should probably also try bonk{-{1,2,3,4,5}} and basically loop over each of these hosts, curl`ing the API, one after another, until the machine is terminated. I'll push up a change.